content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -2875,6 +2875,8 @@ const user = await prisma.profile.create({
 })
 ```
 
+However, you can't use both the direct approach and the `connect` approach in the same query. See [this issue comment](https://github.com/prisma/prisma/issues/4322#issuecomment-737976117) for details.
```

Comment:
Internal note to my colleagues: Can we move the actual information into this sentence to avoid linking to an old issue? The statement from Harshit is still true as far as I know - but might be wort double checking to expand on it.